redeem/Redeem.py

```diff
@@ -73,14 +74,16 @@
 logging.basicConfig(level=logging.DEBUG,
                     format='%(asctime)s %(name)-12s %(levelname)-8s %(message)s',
                     datefmt='%m-%d %H:%M')
+
+
 class Redeem:
     def __init__(self, config_location="/etc/redeem"):
         """
         config_location: provide the location to look for config files.
          - default is installed directory
          - allows for running in a local directory when debugging
         """
-        firmware_version = "2.0.5~Red Heat"
```

Comment:
I'm fine with fixing up the version strings that automation/packaging might see, but can we keep a "friendly version" string around for the logs and for when OctoPrint queries?

Reply:
the line below sets the firmware version to match the version that is installed... i think that is both user friendly and consistent (?)

or maybe I'm misunderstanding what `Red Heat` means? perhaps it's a release candidate signifier? if so, we can set the version to `2.0.5rc1` or if it's alpha quality `2.0.5a1`.

Reply:
"Red Heat" is the friendly name we've assigned to the release based on a tradition Elias started of naming Redeem releases after Schwarzenegger films.

Reply:
What will we do when we reach for more versions than Schwarzenegger movies? Tongue-in-cheek answer would be to start with Van Damme movies. But more seriously what is the strategy there?

Reply:
While I appreciate tradition, I'm not sure it's so user friendly to use names that aren't well understood and well documented. I think it might be time to break from tradition. Can we use RC1 like the rest of the python community does?

Reply:
Eh, who says it has to be ill documented? We can easily put a table on the wiki of the release code names with their versions... I'd suggest a Redeem dev powow of some kind to arrive to a proper agreement on how we go about this. Can we discuss this on Slack instead of here?

Reply:
updated to include a release name for display for easier identification of versions